hack/test/groupsnapshot.storage.k8s.io_volumegroupsnapshotclasses.yaml

```diff
@@ -92,4 +91,4 @@ spec:
         type: object
     served: true
     storage: true
-    subresources: {}
```

Comment:
@BenamarMk , this file is not actually changed?

Reply:
Ok, thanks. I'll remove it.